src/components/Navbar/SideNav.js

```diff
@@ -0,0 +1,81 @@
+import React from 'react';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { faTimes } from '@fortawesome/free-solid-svg-icons';
+import './SideNav.css';
+import { Link } from 'react-router-dom';
+import { Swipeable } from 'react-swipeable';
+import PropTypes from 'prop-types';
+const SideNav = ({
+  navbarOpen,
+  handleToggleMenuClick,
+  handleToggleMenuNavClick,
+}) => {
+  let sideNavClasses = 'SideNav';
+  if (navbarOpen) {
```

Comment:
you can remove this condition and make it like
```suggestion
 const sideNavClasses = navbarOpen 'side-nav open' ? : 'side-nav';
```

also `css` classes shouldn't be capital case